app/electron/main.ts

```diff
@@ -443,6 +443,65 @@ class PluginManagerEventListeners {
   }
 }
 
+/**
+ * Collects bin directories from all installed plugins.
+ * @param {string} pluginsDir - The directory containing plugins
+ * @returns {string[]} Array of plugin bin directory paths
+ */
+function getPluginBinDirectories(pluginsDir: string): string[] {
+  if (!fs.existsSync(pluginsDir)) {
+    return [];
+  }
+
+  const binDirs: string[] = [];
+
+  try {
+    const entries = fs.readdirSync(pluginsDir, { withFileTypes: true });
+    const pluginFolders = entries.filter(entry => entry.isDirectory());
+
+    for (const pluginFolder of pluginFolders) {
+      const binDir = path.join(pluginsDir, pluginFolder.name, 'bin');
+      if (fs.existsSync(binDir)) {
+        // Make sure binaries are executable
+        if (process.platform !== 'win32') {
+          try {
+            const files = fs.readdirSync(binDir);
+            for (const file of files) {
+              const filePath = path.join(binDir, file);
+              // Skip directories
+              if (fs.statSync(filePath).isDirectory()) {
+                continue;
+              }
+              fs.chmodSync(filePath, 0o755); // rwx r-x r-x
+            }
+          } catch (err) {
+            console.error(`Error setting executable permissions in ${binDir}:`, err);
+          }
+        }
+        binDirs.push(binDir);
+      }
+    }
+  } catch (err) {
+    console.error(`Error scanning plugin directories in ${pluginsDir}:`, err);
+  }
+
+  return binDirs;
+}
+
+/**
+ * Adds directories to the PATH environment variable
+ * @param {string[]} dirs - Directories to add to PATH
+ * @param {string} description - Description for logging (e.g., "plugin", "bundled plugin")
+ */
+function addToPath(dirs: string[], description: string): void {
+  if (dirs.length === 0) return;
+
+  const pathSeparator = process.platform === 'win32' ? ';' : ':';
+  const existingPath = process.env.PATH || '';
+  process.env.PATH = [...dirs, existingPath].join(pathSeparator);
```

Comment:
Do we want to use their minikube binary, or the one that comes with the plugin?

There are trade offs for both.

But if someone already has minikube, they may have already configured it, and it would work. Or they might be using a specific version for a reason (bugs in most recent version for example).

On the other hand, they may have downloaded it 4 years ago and never updated it.